<launch>
 
  <node pkg="track_detection" type="Track_Detection_node" name="Track_Detection_node" output="screen">
  	<param name="GyraThreshold" type="int"    	value="60"/>
  	<param name="StartMove"     type="bool"    	value="false"/>
  	<param name="MixKP"     	type="double"   value="0.016"/>
  	<param name="MaxKP"     	type="double"   value="0.019"/>
  	<param name="HiBotSpeed"     type="double"   value="0.2"/>
	</node>


</launch>
